File: py files/3a.py
# 3a: Write a function in Python that greets a user by name.

def greet_user(name):
    """Greets a user by their name"""
    print(f"Hello, {name}! Welcome!")

if __name__ == "__main__":
    user_name = input("Enter your name: ")
    greet_user(user_name)

File: js files/3a.js
// 3a. Write a function in JS that greets a user by name.

function greetUser(name) {
    console.log(`Hello, ${name}! Welcome!`);
}

const name = prompt("Enter your name:");

if (name) {
    greetUser(name);
} else {
    console.log("No name entered.");
}
